### DISCUSSION TITLE
Bats VSCode Extension

### DISCUSSION THREAD

**Discussion by jetmartin** [bats-core/bats-core#825] [Show and tell]
Hi,

I've published while ago a VSC extension for Bats to support one of my dev team.
It's available on [VSC marketplace](https://marketplace.visualstudio.com/items?itemName=jetmartin.bats) and [OpenVSX](https://open-vsx.org/extension/jetmartin/bats).

I've received an update request last month as a core module has been updated.
At the occasion of the update in the store, I'm surprised to see that there are a lots of users (~30 downloads/day).

As I do have very little time to manage my open source activities, I wonder if it will be better to move it to the bats-core organization.
It could also a way to be aware of updates and maintain it properly.

What do you think about it ?

Best.

Jetmartin

**Comment by brokenpip3** [comment]
For what it worth my 2cent: I think that we could embrace it under the org umbrella, I can even offer myself on taking a look on it when I will have all free cycles.
For everyone else: the code is here https://github.com/jetmartin/bats

@bats-core/bats-core wtdy?

**Reply by jetmartin** [reply]
Hi,
Thank you,
Feel free to contact me for any questions/support or for the stores publication.
So after, I will mark extension as migrated on the readme and in the stores.

**Reply by brokenpip3** [reply]
sure, let me talk with the others bats org maintainer and will contact you

**Reply by brokenpip3** [reply]
hey @jetmartin I spoke with @martin-schulze-vireso, we confirmed that it make sense to have it under the org umbrella :)

I'm not 100% sure but I believe you can create a move under this org request from the repo settings, once you do that we can accept and make you collaborator of the repo.. so we can start to maintain it together :)

For sure I will add a PR for the detik syntax/snippet myself

**Reply by jetmartin** [reply]
Hi @brokenpip3,

Happy that you find it valuable ;)
I look at the migration to the org over the WE, I keep you posted here.

Quick question, 
Do you want to move it on the store from my personal publisher to a Bats org publisher ?
(I can create one easily, the deprecation for a new plugin is now supported by MSFT)

Best

**Reply by jetmartin** [reply]
Hi

To be able to transfer the repository to the bats-core org, I need to have the right to create a repo to the org.
<img width="900" alt="image" src="https://github.com/bats-core/bats-core/assets/6155133/52cc1d4a-b88e-4362-ae7d-0f0573a25656">

**Reply by brokenpip3** [reply]
@bats-core/bats-core can someone help here? I can't invite @jetmartin in the bats-core org.
Thanks!

**Reply by martin-schulze-vireso** [reply]
I perused the GitHub documentation and did not find a good solution for this situation. I created a subteam VSCode with the three of us. Can you try if you can move it now @jetmartin?

If not, I am afraid we'll have to find another route, maybe you can give one of us admin rights to the repo, so we can move it?

**Reply by jetmartin** [reply]
Hi,
That's ok, I've just migrated the repo.
https://github.com/bats-core/bats-vscode
I will made minor updates later today to fix links and republished it on MSFT marketplace.

**Reply by jetmartin** [reply]
FYI, No rush to update the repo, Github, manage proper redirections.
@brokenpip3, I will may wait for your PR for the detik syntax/snippet to update the links ;)

**Reply by brokenpip3** [reply]
Nice, welcome in the org :)
> @brokenpip3, I will may wait for your PR for the detik syntax/snippet to update the links ;)

I will, in the mean time can you add a `CONTRIBUTING.md` page where you can explain how to quickly made a change and test it locally so everyone, like me, can contribute to the project?
Thanks!

**Reply by martin-schulze-vireso** [reply]
I am sorry @jetmartin, I had to reinvite you because I misunderstood the team management ui.

**Reply by jetmartin** [reply]
@brokenpip3 , for sure I will, maybe over a WE, I'm busy at work now...
But I assume you're not in a rush for it.

@martin-schulze-vireso , No pb.

**Reply by brokenpip3** [reply]
>But I assume you're not in a rush for it.

Nope, I need to work as well :D
Take your time, there is no urgency in oss ^_^

**Reply by jetmartin** [reply]
HI @brokenpip3,

I had a long flight yesterday wich allow me to spend some time on it.
I've pushed some contrib support.
Please share updates/ask questions as you follow the instructions as I already have a functional setup.

I should also add Github tpls for PR & issues, but I didn't had local templates on my laptop and I forgot the naming conventions.
I'll do it another time.

Lets continue the discussion on the repo instead of spamming here ;)

Best.

**Comment by brokenpip3** [comment]
> HI @brokenpip3,
> 
> I had a long flight yesterday wich allow me to spend some time on it. I've pushed some contrib support. Please share updates/ask questions as you follow the instructions as I already have a functional setup.

Great, ty, will take a look in the following days, it will be a busy month for me but for sure I want to create a PR for bats-detik syntax :)
Let's see
